<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite id="0" name="F:\OrchidOs\__test__\orchidOS.test.js" errors="0" package="F:\OrchidOs\__test__\orchidOS.test.js" hostname="localhost" tests="2" failures="2" time="1.434" timestamp="2024-05-12T02:19:28">
    <properties>
    </properties>
    <testcase classname="OrchidOS HTML structure" name="Header section should exist" time="0">
      <failure message="Error: [2mexpect([22m[31mreceived[39m[2m).[22mtoBeTruthy[2m()[22m

Received: [31mnull[39m
    at Object.toBeTruthy (F:\OrchidOs\__test__\orchidOS.test.js:20:20)
    at Promise.then.completed (F:\OrchidOs\node_modules\jest-circus\build\utils.js:298:28)
    at new Promise (&lt;anonymous&gt;)
    at callAsyncCircusFn (F:\OrchidOs\node_modules\jest-circus\build\utils.js:231:10)
    at _callCircusTest (F:\OrchidOs\node_modules\jest-circus\build\run.js:316:40)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at _runTest (F:\OrchidOs\node_modules\jest-circus\build\run.js:252:3)
    at _runTestsForDescribeBlock (F:\OrchidOs\node_modules\jest-circus\build\run.js:126:9)
    at _runTestsForDescribeBlock (F:\OrchidOs\node_modules\jest-circus\build\run.js:121:9)
    at run (F:\OrchidOs\node_modules\jest-circus\build\run.js:71:3)
    at runAndTransformResultsToJestFormat (F:\OrchidOs\node_modules\jest-circus\build\legacy-code-todo-rewrite\jestAdapterInit.js:122:21)
    at jestAdapter (F:\OrchidOs\node_modules\jest-circus\build\legacy-code-todo-rewrite\jestAdapter.js:79:19)
    at runTestInternal (F:\OrchidOs\node_modules\jest-runner\build\runTest.js:367:16)
    at runTest (F:\OrchidOs\node_modules\jest-runner\build\runTest.js:444:34)" type="AssertionError">
      </failure>
    </testcase>
    <testcase classname="OrchidOS HTML structure" name="Footer section should contain social links and copyright information" time="0">
      <failure message="Error: [2mexpect([22m[31mreceived[39m[2m).[22mtoBeTruthy[2m()[22m

Received: [31mnull[39m
    at Object.toBeTruthy (F:\OrchidOs\__test__\orchidOS.test.js:26:20)
    at Promise.then.completed (F:\OrchidOs\node_modules\jest-circus\build\utils.js:298:28)
    at new Promise (&lt;anonymous&gt;)
    at callAsyncCircusFn (F:\OrchidOs\node_modules\jest-circus\build\utils.js:231:10)
    at _callCircusTest (F:\OrchidOs\node_modules\jest-circus\build\run.js:316:40)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at _runTest (F:\OrchidOs\node_modules\jest-circus\build\run.js:252:3)
    at _runTestsForDescribeBlock (F:\OrchidOs\node_modules\jest-circus\build\run.js:126:9)
    at _runTestsForDescribeBlock (F:\OrchidOs\node_modules\jest-circus\build\run.js:121:9)
    at run (F:\OrchidOs\node_modules\jest-circus\build\run.js:71:3)
    at runAndTransformResultsToJestFormat (F:\OrchidOs\node_modules\jest-circus\build\legacy-code-todo-rewrite\jestAdapterInit.js:122:21)
    at jestAdapter (F:\OrchidOs\node_modules\jest-circus\build\legacy-code-todo-rewrite\jestAdapter.js:79:19)
    at runTestInternal (F:\OrchidOs\node_modules\jest-runner\build\runTest.js:367:16)
    at runTest (F:\OrchidOs\node_modules\jest-runner\build\runTest.js:444:34)" type="AssertionError">
      </failure>
    </testcase>
    <system-out/>
    <system-err/>
  </testsuite>
</testsuites>